tools/shared/pragma.c

```diff
@@ -1184,37 +1184,27 @@ do_sw(void)
       return true;
     break;
   case SW_UNROLL:
-    /* [no]unroll		-x/y 11 3
-     * [no]unroll = c	-x/y 11 1
-     * [no]unroll = n	-x/y 11 2
-     *     unroll = c:v     -y   11 3  -x    9 v
-     *     unroll = n:v     -y   11 3  -x   10 v
+    /* unroll       -x 11 0x1   -y 11 0x402
+     * unroll = n   -x 11 0x2   -y 11 0x401 -x 9 n
+     * unroll (n)   -x 11 0x2   -y 11 0x401 -x 9 n
```

Comment:
Is the 401, 402 setting used anywhere?

Reply:
AFAIK, this is the only use of bits 0x400, 0x2, and 0x1, in x-flag 11. #742 used 0x400 to mean `NOUNROLL`, and this PR repurposes 0x2 to mean `UNROLL(n)`. The original distinction between 0x1 and 0x2 did not seem applicable to the LLVM loop unrolling parameters.